### ISSUE TITLE
Single Frame (No Screen Display) PDF broken

### ISSUE THREAD

**Issue by shiffman** [processing/processing#3280]
In 3.0a7, the example from the reference is not working:

https://processing.org/reference/libraries/pdf/
```
import processing.pdf.*;

void setup() {
  size(400, 400, PDF, "filename.pdf");
}

void draw() {
  // Draw something good here
  line(0, 0, width/2, height);

  // Exit the program 
  println("Finished.");
  exit();
}
```

```
Because you're not running from the PDE, add this to your code:
public String sketchRenderer() {
  return "processing.pdf.PGraphicsPDF";
}
```

Adding the requested method results in:

```
java.lang.NullPointerException
	at processing.core.PSurfaceNone$AnimationThread.run(PSurfaceNone.java:230)
```

Apologies if this is known as I know renderer work is in progress.  I'll be testing other examples today and will follow-up in this thread.

**Comment by shiffman** [comment]
"Single Frame (With Screen Display)" does, in fact, work correctly.

**Comment by benfry** [comment]
Ugh, ok.

**Comment by benfry** [comment]
Interesting, this is actually a bug in how `size()` is parsed. It's just not shown up until now, when we're getting pickier about how the renderer is specified.

**Comment by benfry** [comment]
Fixed for 3.0a8.
